### ISSUE TITLE
Pramda - Practical Functional Programming in PHP

### ISSUE THREAD

**Issue by kapolos** [thechangelog/ping#352]
https://github.com/kapolos/pramda

* Automatically Curried Functions
* Lazy Evaluation baked-in
* Fun(ctional) to Use

**Comment by jerodsanto** [comment]
@kapolos I love your ELI5 on FP in the readme. I just queued up a [tweet](https://twitter.com/changelog) in your direction and we'll link Pramda up in our next issue of [Changelog Weekly](https://changelog.com/weekly/). It :ship:s tomorrow morning. :beers:

**Comment by kapolos** [comment]
Thank you @jerodsanto! :+1: Very glad to hear you like it :beers:
